src/ui/css/vars.scss

```diff
@@ -1,65 +0,0 @@
-@import '~amo/css/inc/vars';
-
-// Desktop variables (from specs)
-$black: #343a40;
-$white: #fff;
-$header-base-color: #0f1126;
-$footer-base-color: #051818;
-$header-search-color: #36395b;
-$action-base-color: #329af0;
-$action-hover-color: #72c3fc;
-$action-active-color: #1c7cd6;
-$cancel-base-color: $grey-50;
-$cancel-hover-color: $grey-40;
-$cancel-active-color: $grey-50;
-$confirm-base-color: #51cf66;
-$confirm-hover-color: #8ce99a;
-$confirm-active-color: #37b24d;
-$alert-base-color: #fcc419;
-$alert-hover-color: #ffe066;
-$alert-active-color: #f59f00;
-$report-base-color: #fa5252;
-$report-hover-color: #ff8787;
-$report-active-color: #e03131;
-$neutral-base-color: #adb5bd;
-$neutral-hover-color: #dee2e6;
-$neutral-active-color: #868e96;
-$light-base-color: $white;
-$light-hover-color: darken($white, 15%);
-$light-active-color: darken($white, 5%);
-$block-base-white-color: $white;
-$block-hover-white-color: #f8f9fa;
-$background-grey: #e9ecef;
-$background-gray: $background-grey;
-$type-black: $black;
-$header-accent-color: $link-color;
-$header-text-not-active-color: transparentize($white, 0.5);
-
-// Category Colors
-// There are currently 12 colors and this number must be kept in sync with
-// `src/amo/components/Categories/index.js`.
-$category-colors: $teal-70, $red-60, $magenta-60, $orange-60, $blue-50,
-  $green-70, $magenta-70, $red-50, $purple-40, $orange-70, $blue-60, $yellow-70;
-
-// Fonts
-$font-size-s: 12px;
-$font-size-m-smaller: 14px;
-$font-size-m: 18px;
```

Comment:
This is not consistent with what we have in the `amo` css... Other values are correct so now former "UI" components will use the `size-m` of the `amo` css (which is 16px).